browser/widevine/widevine_prefs_migration_browsertest.cc

```diff
@@ -0,0 +1,23 @@
+/* Copyright (c) 2019 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/common/pref_names.h"
+#include "chrome/browser/browser_process.h"
+#include "chrome/test/base/in_process_browser_test.h"
+#include "components/prefs/pref_service.h"
+#include "third_party/widevine/cdm/buildflags.h"
+
+using WidevinePrefsMigrationTest = InProcessBrowserTest;
+
+IN_PROC_BROWSER_TEST_F(WidevinePrefsMigrationTest, PrefMigrationTest) {
```

Comment:
Is it possible to add a `PRE_PrefMigrationTest` and set local_state to default value and also set profile value for `kWidevineOptedIn` and `kWidevineInstalledVersion` to verify the migrated values?

Reply:
I tried that scenario at first.
But, local_state is not the default value at the second launching.(Of course, cleared local state in the first launching).
Not sure why it happens. I'm trying again.

Reply:
Hmm, local state seems not consistent during the spanning tests.
In the second launching, it's state is not the last state of first browser launching.
profile prefs shows consistent state but not local state during the spanning test.

Reply:
local state is persisted but not persisted after clearing prefs(`ClearPref()`).
At second test, local state has the value before the clearing. I expected it has default value because `ClearePref()` is called but it's not default value at second test.
So, testing is done by explicitly calling `MigrateWidevinePrefs()` instead of spanning test.